aws/logs_monitoring/caching/cloudwatch_log_group_cache.py

```diff
@@ -67,29 +63,6 @@ def get(self, log_group_arn):
     def _should_fetch_tags(self):
         return os.environ.get("DD_FETCH_LOG_GROUP_TAGS", "false").lower() == "true"
 
-    def _build_tags_cache(self):
```

Comment:
nit: you may want to keep this code there and just remove its usage above

Reply:
It's not called somewhere else, it will be a dead code. If we want to revert it, it's preferable to revert the commit I think.